Makefile.am

```diff
@@ -0,0 +1,144 @@
+SUBDIRS = cppForSwig
```

Comment:
Automake/Makefile targets and dependencies aren't really being used here at all? I'm not sure what the advantage of using Make is.